cluster/gce/addons/apiserver-network-proxy/daemonset.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+    k8s-app: proxy-agent
+  namespace: kube-system
+  name: proxy-agent
```

Comment:
This is making an already overloaded name even more overloaded. We already have a "kube-proxy" and multiple API paths that involve a proxy of some sort. I have the same comment on the server half. 
* cp2n-connectivity-agent & cp2n-connectivity-server (i.e., control plane to node)
* tako-agent and tako-server?
* ?

Reply:
konnectivity-server and konnectivity-agent.

Reply:
Fixed